src/assets/global.scss

```diff
@@ -1,4 +1,5 @@
 @import './variables';
+@import './raleway';
```

Comment:
I Agree with you, I personally prefer Roboto, but it's written in the kuzzle graphic chard that we have to use Raleway ( :cry: )